posts/Information-as-an-Asset-Through-Business-Intelligence-and-Data-Analytics.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Can you rename the file to lowercase?

--> `information-as-an-asset-through-business-intelligence-and-data-analytics.md`